benchmark/events/ee-emit-multi-args.js

```diff
@@ -0,0 +1,20 @@
+var common = require('../common.js');
+var EventEmitter = require('events').EventEmitter;
+ 
+var bench = common.createBenchmark(main, {n: [25e4]});
+ 
+function main(conf) {
+  var n = conf.n | 0;
+ 
+  var ee = new EventEmitter();
+  var listeners = [];
+ 
+  for (var k = 0; k < 10; k += 1)
+    ee.on('dummy', function() {});
+ 
+  bench.start();
+  for (var i = 0; i < n; i += 1) {
+    ee.emit('dummy', 5, true);
```

Comment:
I was just thinking on the basis of your [PR comment](https://github.com/iojs/io.js/pull/601#issuecomment-72066349), and that it would be best to express that in a benchmark -- of course, this comment is pretty much only based on that PR, so thats up to you